rust/main/chains/hyperlane-ethereum/src/signer/singleton.rs

```diff
@@ -38,6 +38,14 @@ pub struct SingletonSignerHandle {
     tx: mpsc::UnboundedSender<SignTask>,
 }
 
+#[cfg(feature = "test-utils")]
```

Comment:
since there was no constructor method available previously, I want to preserve any intended encapsulation so this is only available to build mock test structs